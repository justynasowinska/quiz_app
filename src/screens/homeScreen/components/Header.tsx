import * as React from 'react';
import { Text, StyleSheet } from 'react-native';

import { THEME_COLORS } from '../../../config/colors';

export const Header = () => {
    return (
        <Text style={styles.header}>
            Welcome to the{'\n'}Trivia Challenge!
        </Text>
    );
};

const styles = StyleSheet.create({
    header: {
        fontSize: 26,
        fontWeight: 'bold',
        color: THEME_COLORS.headerTextColor,
        textAlign: 'center'
    }
});