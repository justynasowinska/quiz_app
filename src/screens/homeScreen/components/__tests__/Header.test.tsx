import * as React from 'react';
import { shallow } from 'enzyme';

import { Header } from '../Header';

describe('Home Screen Header Component', () => {
    it('matches to the snapshot', () => {
        const component = shallow(<Header />);

        expect(component).toMatchSnapshot();
    });
});
