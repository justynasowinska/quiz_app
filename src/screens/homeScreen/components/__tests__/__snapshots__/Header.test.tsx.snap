// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen Header Component matches to the snapshot 1`] = `
<Text
  style={
    Object {
      "color": "#0C9198",
      "fontSize": 26,
      "fontWeight": "bold",
      "textAlign": "center",
    }
  }
>
  Welcome to the
  

  Trivia Challenge!
</Text>
`;
